eth/tracers/tracer.go

```diff
@@ -413,9 +415,16 @@ func New(code string, ctx *Context) (*Tracer, error) {
 		copy(makeSlice(ctx.PushFixedBuffer(20), 20), contract[:])
 		return 1
 	})
+	tracer.activePrecompiles = vm.PrecompiledAddressesHomestead
```

Comment:
This is a mistake, right?

Reply:
Ah, ok, I see, you set them just to whatever, and then init them properly in `CaptureStart`. Yeah I can see the reasoning behind that, but OTOH you might just also leave the slice empty. Both are "wrong" but, the latter case is more obviously wrong, not just "nearly correct but not quite" (which typically makes it difficult to find bugs)

Reply:
Yeah exactly just wanted to avoid having an empty slice. But I get your point. Fixed